cpp/src/util/filters.cu

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cuvs/core/bitmap.hpp>
+#include <cuvs/core/bitset.hpp>
+#include <faiss/impl/IDSelector.h>
```

Comment:
I don't think we want to be using faiss types in cuVS. That would introduce a circular dependency. Instead, we should implement the conversion logic in FAISS itself.